Adam_Dochtorowicz_Latex_HW

```diff
@@ -0,0 +1,127 @@
+\documentclass{article}
+\usepackage[utf8]{inputenc}
+\usepackage{tikz}
+\usepackage{phonrule}
+\usepackage{tikz-qtree}
+\usepackage{tipa}
+\usepackage{amsmath}
+\usepackage{lipsum}
+\usepackage[margin=2.54cm, left=3.81cm, includefoot]{geometry}
+\usepackage{graphicx}
+\usepackage{float}
+\usepackage{pgfplots}
+\usepackage{setspace}
+
+\usepackage{fancyhdr}
+\pagestyle{fancy}
+\fancyhead{}
+\fancyfoot{}
+\fancyfoot[R]{\thepage}
+\renewcommand{\headrulewidth}{0pt}
+\renewcommand{\footrulewidth}{2pt}
+
+\begin{document}
+
+\begin{titlepage}
+    \begin{center}
+    \line(1,0){300}\\
+    [5mm]
+    \huge{\bfseries LaTeX HW}\\
+    [2mm]
+    \line(1,0){250}\\
+    [1.5mm]
+    \textsc{\LARGE Gdansk University}\\
+    [2mm]
+    \textsc{\Large Different functions of LaTeX}\\
+    [13cm]
+    \end{center}
+    \begin{flushright}
+    \textsc{\large Adam Dochtorowicz}\\
+    NLP student\\
+    \# 234564\\
+    June 11, 2019
+    \end{flushright}
+\end{titlepage}
+\cleardoublepage
+
+\tableofcontents
+\thispagestyle{empty}
+\cleardoublepage
+
+\section{Favorite topics}
+\subsection{Linguistics}
+\begin{itemize}
+    \item Phonology
+    \item Phonetics
+    \item Semantics
+    \item Sociolinguistics
+    \item Forensic Linguistics
+\end{itemize}
+
+\subsection{Computer Science}
+\begin{itemize}
+    \item Machine Learning
+    \item Data Science
+    \item AI
+\end{itemize}
+
+\section{\textipa{f@"nEtIk tr\ae nskrIpS@n}}
+\subsection{\textipa{"INglIS tr\ae nskrIpS@n}}
+\textipa{| haI maI neIm Iz "\ae d@m | aI T\super jINk Its f2n t@ "k\super h@Ud In "leI""tEk | "k\super h3r@ntli aIm "st2dIN @t D@ ""jun@"v3rs@ti @v "gda\textltailn sk | In maI fri taIm aI laIk t@ sw\super jIm @nd r2n In maI "t\super haUn} 
+\subsection{\textipa{ "p\super h@UlIS ""tr\ae n"skrIpS@n  tr\~a\~ws"kr1p\texttslig\super jja "pOlska}}
+\textipa{| lub\super jE \textctc \~E \ | dObZe bav\super ji\texttctclig | v S\textteshlig EgulnoC\texttctclig i gd1 jEstEm sam v klub\super jie| natOm\super jiast gd1 jEstEm v \textctc rud lu\textctdctzlig i zahovujE \textctc \~E bar\textdzlig O gZE\textteshlig \textltailn\super jE }
```

Comment:
1. w (klubie, śród) - devoicing! /f/!
2. No palatalization after a palatal nasal
3. klubie, natomiast - do we have the /i/ phoneme here?
4. zachowuję - /x/ instead of /h/
5. remember about final devoicing